test.lua

```diff
@@ -470,4 +470,16 @@ describe('mpack', function()
       assert.are_same("mpack.NIL", tostring(mpack.NIL))
     end)
   end)
+
+  it('should allow to be required twice', function()
```

Comment:
nit:  less awkward wording:

```suggestion
  it('can be required twice', function()
```